bears/typescript/TSLintBear.py

```diff
@@ -37,7 +37,7 @@ def create_arguments(filename, file, config_file,
         args = ('--format', 'json')
         if tslint_config:
             args += ('--config', tslint_config)
-        if rules_dir:
+        if rules_dir:  # pragma: no cover
```

Comment:
Worth noting that since tslint requires a `tslint.json` otherwise it does nothing if no rules are selected , `rules_dir` is a bit redundant as `tslint.json` will usually contain the `rulesDirectory`.

It might be simpler to deprecated/decommission rules_dir, or at least not enhance it until someone does some analysis to show it is useful.

Reply:
If we don't provide `rules_dir` option in the future, that means `tslint_config` option becomes a must?

Reply:
If there is no value for `tslint_config` , tslint may find tslint.json in the top level directory where coala is run.  If there is no tslint.json, there are no rules being used, especially not any custom rules which might reside in a custom directory.

If there is a `tslint.json`, why wouldnt it include `rulesDirectory` ?  It seems silly to not use `rulesDirectory`.

It would seem to me that a `rules_dir` setting only makes sense if the bear also had a setting for custom rule names to enforce, in which case a directory to get the rules from is needed.

But more analysis may expose a reason for `rules_dir`.